portafly/src/components/pages/accounts/utils/accountsTableDataFactory.tsx

```diff
@@ -2,9 +2,10 @@ import React from 'react'
 import { sortable } from '@patternfly/react-table'
 import { IDeveloperAccount } from 'types'
 import { TFunction } from 'i18next'
-import { ActionButtonImpersonate } from 'components'
+import { ActionButtonImpersonate } from 'components/pages/accounts'
 
-const generateRows = (accounts: IDeveloperAccount[], isMultitenant = false) => {
+const generateRows = (accounts: IDeveloperAccount[]) => {
+  const isMultitenant = process.env.REACT_APP_MULTITENANT === 'true'
```

Comment:
```suggestion
  const isMultitenant = process.env.REACT_APP_MULTITENANT
```

Reply:
The problem I see with this is if I wrongly set my var to `REACT_APP_MULTITENANT="fals"` it's going to be read as `true`.

Reply:
That problem would denote a poorly setup/orchestration. Specially when this will be managed by OpenShift or any other K8s solution